### PR TITLE
Sage font updates

### PR DESCRIPTION
**Pull request by joedakroub**
## Description
This PR adds/udpates the following icons:

### New and redesigned icons:

- file
- draft
- duplicate 
- rename 
- location

### Fixes for icons that didn't have a rounded edge:

- closed-captions 
- add-image 
- flag 
- danger

## Test notes
Open http://0.0.0.0:4000/pages/design/icon and ensure each icon matches its name.

### CODE REVIEW

File: docs/app/views/pages/icon.html.erb [hunk 1]
```diff
@@ -198,7 +199,8 @@
   "circle-7",
   "circle-8",
   "circle-9",
-  "clock",
+  #"clock",
```
Comment (philschanely) [inline docs/app/views/pages/icon.html.erb:202]: @joedakroub is this an accident? Looks like this one is included in lists elsewhere.

**Review by philschanely — APPROVED** [review]
One small question. 👍

**Review by monkeypox8 — COMMENTED** [review]
Looks like the outlines on the following items were filled in the export:
- help
- preview-off
- success
- user-circle
- send-message
- color
- pen

icon-help
icon-preview-off
icon-send-message
icon-success
icon-user-circle
icon-color
icon-pen

File: docs/app/views/pages/icon.html.erb (hunk 1 above)
Comment (joedakroub) [reply docs/app/views/pages/icon.html.erb]: @philschanely Ah, yes that is an accident. Thank you!

**Review by QuintonJason — APPROVED** [review]

**Comment by joedakroub** [comment]
I've updated the Sage icon font with the latest from design. Please ensure that the icons are rendering properly.

**Review by monkeypox8 — APPROVED** [review]
👍🏼 LGTM, thanks for wrangling this one!

**Review by philschanely — APPROVED** [review]
🙌

**Review by QuintonJason — APPROVED** [review]
💥

**Pull request merged**